### DISCUSSION TITLE
ability to reset useNameserver

### DISCUSSION THREAD

**Discussion by plandolt** [spatie/dns#97] [Ideas]
The current implementation prevents to clear the `useNameserver` function. What about allow `null` for `useNameserver` or introduce a `cleareNameserver` function?

**Comment by freekmurze** [comment]
What would be the use case? Feel free to PR improvements, Make sure to include updated tests / readme.

**Comment by plandolt** [comment]
In the meantime, my use case changed. The code acts like a resolver finding the authoritative nameservers starting from the root nameservers and thus will always use a specific nameserver. If the need for me or another use arises again, I'm gladly creating a PR.
